enterprise/cli/exp_aibridge.go

```diff
@@ -0,0 +1,153 @@
+package cli
+
+import (
+	"encoding/json"
+	"math"
+	"time"
+
+	"github.com/google/uuid"
+	"golang.org/x/xerrors"
+
+	"github.com/coder/coder/v2/codersdk"
+	"github.com/coder/serpent"
+)
+
+func (r *RootCmd) aibridge() *serpent.Command {
+	cmd := &serpent.Command{
+		Use:   "aibridge",
+		Short: "Manage AIBridge.",
+		Handler: func(inv *serpent.Invocation) error {
+			return inv.Command.HelpHandler(inv)
+		},
+		Children: []*serpent.Command{
+			r.listInterceptions(),
+		},
+	}
+	return cmd
+}
+
+func (r *RootCmd) listInterceptions() *serpent.Command {
+	var (
+		initiator        string
+		startedBeforeRaw string
+		startedAfterRaw  string
+		provider         string
+		model            string
+		afterIDRaw       string
+		limit            int64
+	)
+
+	return &serpent.Command{
+		Use:   "list-interceptions",
+		Short: "List AIBridge interceptions as JSON.",
+		Options: serpent.OptionSet{
+			{
+				Flag:        "initiator",
+				Description: `Only return interceptions initiated by this user. Accepts a user ID, username, or "me".`,
+				Default:     "",
+				Value:       serpent.StringOf(&initiator),
+			},
+			{
+				Flag:        "started-before",
+				Description: `Only return interceptions started before this time. Accepts a time in the format "2021-01-01T00:00:00Z".`,
```

Comment:
Nit: let's use the Go date format which we return in errors to keep things consistent (`2006-01-02T15:04:05Z07:00`) and specify it's RFC 3339.